README.md

```diff
@@ -336,20 +361,13 @@ The argument to this method can either be a simple string indicating
 the target for messages of interest (e.g. a queue name), or an options
 object that may contain any of the following fields:
 
-  * target - The target to which messages are sent. This can be a
-    simple string address/name or a nested object itself containing
-    the fields:
-    * address
-    * dynamic
-    * expiry_policy
-    * durable
-  * source - The source of a sending link is the local identifier. It
-    is usually not needed, but can be set if it is,
   * name - The name of the link. This should be unique for the
     container. If not specified a unqiue name is generated.
   * autosettle - Whether sent messages should be automatically
     settled once the peer settles them. Defaults to true.
 
+And attach frame fields as for `open_receiver`.
```

Comment:
Source and target are used differently for senders and receivers, so should be pulled out of the common set or the descriptions are wrong for senders.

Reply:
True.